resources/monitoring/values.yaml

```diff
@@ -105,18 +120,29 @@ defaultRules:
   ## Annotations for default rules
   annotations: {}
 
-## Provide custom recording or alerting rules to be deployed into the cluster.
+  ## Additional labels for PrometheusRule alerts
+  additionalRuleLabels: {}
+
+## Deprecated way to provide custom recording or alerting rules to be deployed into the cluster.
 ##
-additionalPrometheusRules: |-
```

Comment:
I think we should keep it as is, right? @a-thaler

Reply:
Used to be empty (or rather a comment block) and was deprecated in favour of additionalPrometheusRulesMap (see below)